configure.sh

```diff
@@ -62,6 +62,21 @@ else
 fi
 popd > /dev/null
 
+pushd ${MY_DIR}
+if command -v cmake &> /dev/null
+then
+    echo "CMake is installed"
+else
+    echo "CMake is not installed"
+    wget https://github.com/Kitware/CMake/archive/refs/tags/v3.30.0.zip -P host-tools/. --no-check-certificate
+    cd host-tools
+    unzip v3.30.0.zip
+    cd ${CMAKE_DIR}
+    mkdir build && cd build
+    ../bootstrap --prefix=./. && make -j4 BUILD_TESTING=OFF BUILD_EXAMPLES=OFF && make install
```

Comment:
does `make install ` use the --prefix to set the directory to install cmake too? you need to ensure that it doesn't try to install in `/bin/` or something.

Reply:
no, it doesn't install in /bin/. As I have provided the prefix here:  `../bootstrap --prefix=./.`
If it was installing there, it will ask for root/sudo permission which its not. So, I am assuming its all good.